.gitignore

```diff
@@ -2,6 +2,7 @@ tmp
 compile_commands.json
 .elixir_ls
 .gdb_history
+priv/
```

Comment:
Can you check whether this is necessary?

Reply:
Bc from what I understand this is not in our standard `.gitignore`

Reply:
I had updated dialyzer config and removed it